examples/azure-linux-tdx-ita-attestation-vm/variables.tf

```diff
@@ -0,0 +1,64 @@
+############################################################################################
+# Replace the default values with your own values - your trustauthority_api_key (token)    #
+############################################################################################
+# Variable for trustauthority_api_key
+variable "trustauthority_api_key" {
+  description = "trustauthority_api_key"
+    type        = string
+    default     = "<YOUR ITA TOKEN HERE>"
```

Comment:
comment out to ensure user has to pass a value

Reply:
Done